docs: CRUD support for Secrets in Snow (#201)
localstack/localstack-docs: src/content/docs/snowflake/features/secrets.md

---
title: Secrets
description: Get started with Secrets in LocalStack for Snowflake
tags: ["Base"]
---

## Introduction
Secrets in Snowflake provide a secure way to store sensitive credentials, such as usernames and passwords, for use with external integrations. They allow you to centralize authentication information and manage access consistently across your Snowflake environment. 

> The Snowflake emulator offers CRUD support, not the actual implementation or usage of them.  This makes it possible to test workloads locally that rely on secure credential management without needing a live Snowflake account.

Author: ```suggestion
The Snowflake emulator offers CRUD support,  which are currently mocked and not functional.  This makes it possible to test workloads locally that rely on secure credential management without needing a live Snowflake account.
```
